packages/relay/src/lib/services/metricService/metricService.ts

```diff
@@ -220,6 +275,20 @@ export default class MetricService {
     this.captureMetrics(executionMode, txConstructorName, status, cost, gasUsed, callerName, interactingEntity);
   };
 
+  public incrementConsensusClientDurationResets = ({ duration }: IExecuteConsenusClientResetDurationPayload): void => {
+    this.consensusClientResetsDuration.inc(1);
+  };
+
+  public incrementConsensusClientErrorResets = ({ errorCodes }: IExecuteConsenusClientResetErrorPayload): void => {
+    this.consensusClientResetsError.inc(1);
+  };
+
+  public incrementConsensusClientTransactionResets = ({
```

Comment:
let's try to include JSDocs for these methods

Reply:
Done.